hash/gbcolor.xml

```diff
@@ -21541,14 +22551,15 @@ Unreleased (music source code exists, possibly no prototypes exist)
 	</software>
 
 	<software name="tsurisn2">
-		<!-- Notes: SGB enhanced -->
 		<description>Tsuri Sensei 2 (Jpn)</description>
 		<year>1999</year>
 		<publisher>J-Wing</publisher>
 		<info name="serial" value="DMG-AF2J-JPN"/>
+		<info name="gold" value="19990624"/>
 		<info name="release" value="19990723"/>
-		<info name="alt_title" value="昆虫博士2"/>
+		<info name="alt_title" value="昆虫博士 ２"/>
```

Comment:
Japanese sources use a half-width “2” and no space, e.g. https://w.atwiki.jp/gcmatome/pages/6173.html

Reply:
I just got that from the official spreadsheet, like all the others.

Reply:
I've reverted the change.